src/main/java/de/zalando/aruha/nakadi/domain/EventTypeStatistics.java

```diff
@@ -0,0 +1,73 @@
+package de.zalando.aruha.nakadi.domain;
+
+import java.util.Objects;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+public class EventTypeStatistics {
+    @NotNull
+    @Min(value = 0, message = "expected_write_rate can't be less then zero")
+    private Integer expectedWriteRate; // messages per minute
+    @NotNull
+    @Min(value = 1, message = "message_size can't be less then 1")
+    private Integer messageSize; // bytes
+    @NotNull
+    @Min(value = 1, message = "problem with read_parallelism: at least one reader expected")
```

Comment:
Is it necessary to state the field name where there was a problem? I think this is already present by default in the final json problem message. In other places in the code we don't have this, for example `EventType.name` message is `format not allowed`.